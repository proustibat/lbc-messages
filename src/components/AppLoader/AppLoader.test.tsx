import React from 'react';
import { render } from "@testing-library/react";
import { AppLoader } from './index';

describe('AppLoader', () => {
  it('should render correctly by default', () => {
    // Given / When
    const { container } = render(<AppLoader />);

    // Then
    expect(container).toMatchSnapshot();
  });
});
