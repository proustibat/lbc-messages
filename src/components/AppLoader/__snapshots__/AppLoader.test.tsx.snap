// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppLoader should render correctly by default 1`] = `
<div>
  <div
    aria-hidden="true"
    class="MuiBackdrop-root backdrop_zIndex-1201 backdrop_color-#fff"
    style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
  >
    <div
      class="MuiCircularProgress-root MuiCircularProgress-indeterminate"
      role="progressbar"
      style="width: 40px; height: 40px;"
    >
      <svg
        class="MuiCircularProgress-svg"
        viewBox="22 22 44 44"
      >
        <circle
          class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
          cx="44"
          cy="44"
          fill="none"
          r="20.2"
          stroke-width="3.6"
        />
      </svg>
    </div>
  </div>
</div>
`;
